samples/net/sockets/http_get/src/wifi.c

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
All the wifi connection related activity can be done from wifi shell, why to have it here too?

Reply:
This is to follow `samples/subsys/mgmt/updatehub` or `samples/net/cloud/tagoio_http_post`. Though wifi shell has the same wifi connectivity logic, samples may get complicated after integration with wifi shell.

Reply:
This is exactly my point. We should try to concentrate the wifi connectivity to one place instead of having it implemented in many places.
Imho, integrating wifi shell with your app should be quite simple, for example one can execute shell command from app using this function `int shell_execute_cmd(const struct shell *shell, const char *cmd);`. Application just needs to create a string with proper params and then call this function.